packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/interactioncode/use-flow.md

```diff
@@ -1,26 +1,26 @@
-The Interaction Code flow is intended for developers who want to control the remediation user experience without redirecting the authentication experience to Okta. The Interaction Code flow is used in the [embedded authentication deployment model](/docs/concepts/redirect-vs-embedded/#embedded-authentication) where you can manage the user interaction by leveraging either an Identity Engine SDK or the Sign-In Widget with an Identity Engine SDK.
+The Interaction Code flow is intended for developers who want to embed the authentication process in their apps rather than redirect it to Okta. Implement it in the [embedded authentication deployment model](/docs/concepts/redirect-vs-embedded/#embedded-authentication) using an Identity Engine SDK (IDX SDK) either alone or with the Sign-In Widget.
```

Comment:
We can't use IDX as an abbreviation - it was never approved to use in public-facing docs as an abbreviation. Can you please change them all back? Thanks.
```suggestion
The Interaction Code flow is intended for developers who want to embed the authentication process in their apps rather than redirect it to Okta. Implement it in the [embedded authentication deployment model](/docs/concepts/redirect-vs-embedded/#embedded-authentication) using an Identity Engine SDK either alone or with the Sign-In Widget.
```

Reply:
I didn't know that. Will make a note in the Acronyms list. 
Someone should tell all SDK Readme owners :-)